### ISSUE TITLE
The theory of resizer？

### ISSUE THREAD

**Issue by RaySue** [lukas-blecher/LaTeX-OCR#108]
Why need to train the resizer? How much difference betweed Using resizer and Not using？ Thx.

**Comment by lukas-blecher** [comment]
The training data I generated has a very limited resolution range. Most images are no higher than 32px, but when you capture a snippet of your screen the resulting image often has a much higher resolution due to the high pixel density of modern displays.
Using the resizer model you can scale the image down to a resolution that best fits into the training domain. 
That way you get the best accuracy.
Of course, This step is not necessary if you evaluate on the test set generated from the same distribution.

**Comment by RaySue** [comment]
Good tricks!  is there some papers or methods support the resizer？ or the resizer is you first proposed？

**Comment by lukas-blecher** [comment]
I did this on my own. 
Maybe I'm not the first, I didn't check existing papers because the first thing I tried worked just fine.
